packages/shared/src/components/post/SquadPostPageNavigation.tsx

```diff
@@ -29,6 +33,11 @@ function SquadPostPageNavigation({
 }: SquadPostPageNavigationProps): ReactElement {
   const isFromNavigation = checkIsFromNavigation();
   const link = isFromNavigation ? NOTIFICATIONS_LINK : squadLink;
+  const layout = useFeature(feature.feedLayout);
```

Comment:
Any reason here for not using the useFeedLayout?

Reply:
Using the useFeedLayout would always result in `false` since there is no context of feed on the post page.